pkg/postgres/walker/schema.go

```diff
@@ -65,6 +65,9 @@ type SchemaRelationship struct {
 	// NoConstraint indicates that this relationship is not enforced at the SQL
 	// level by a foreign key constraint.
 	NoConstraint bool
+
+	// DeleteRestrict indicates that this relationship should restrict deletion
+	DeleteRestrict bool
```

Comment:
Nit: I would flip the naming here. RestrictDelete seems more apt

Reply:
Renamed the field to bring it in line with this and reverified that the behavior is the same